packages/trail-core/lib/index.js

```diff
@@ -171,6 +171,7 @@ class TrailsManager {
   async get (id) {
     const sql = SQL`
       SELECT
+          id::int,
```

Comment:
In this test:
https://github.com/nearform/trail/blob/bb53ffc44932f32b03de936e90c9721012f67884/packages/trail-hapi-plugin/test/routes/trails.test.js#L196-L232

1. we create a trail using `POST /trails`, and expect a full trail in return
2. we delete the trail.

Unfortunately, `request.trailCore.get()` does not return id, and the delete operation is invoked with undefined.

Interestingly, this does not fail locally, on CI [complains](https://travis-ci.org/nearform/trail/jobs/536806111#L603). 
I guess this is a different version of SQL template lib.